packages/devtools_app/lib/src/device_dialog.dart

```diff
@@ -254,7 +254,7 @@ class _NameColumn extends ColumnData<_DialogFlag> {
 }
 
 class _DescriptionColumn extends ColumnData<_DialogFlag> {
-  _DescriptionColumn() : super.wide('Description');
+  _DescriptionColumn() : super.wide('Description', minWidthPx: 100);
```

Comment:
probably not useful to shrink the description down to zero width so give it a min width.